CMakeLists.txt

```diff
@@ -120,23 +120,23 @@ find_package2(PUBLIC CUDA)
 find_package2(PUBLIC HiRAEVT CONFIG)
 find_package2(PUBLIC GENFIT2)
 
-find_package2(PUBLIC Eigen3 3.3 REQUIRED NO_MODULE)
+#find_package2(PUBLIC Eigen3 3.3 REQUIRED NO_MODULE)
```

Comment:
It's probably better to make this line `find_package2(PUBLIC Eigen3 3.3 NO_MODULE)`